Sources/XMLCoder/Auxiliaries/XMLElement.swift

```diff
@@ -0,0 +1,280 @@
+//
+//  XMLElement.swift
+//  XMLCoder
+//
+//  Created by Vincent Esche on 12/18/18.
+//
+
+import Foundation
+
+internal class _XMLElement {
+    static let attributesKey = "___ATTRIBUTES"
+    static let escapedCharacterSet = [("&", "&amp"), ("<", "&lt;"), (">", "&gt;"), ("'", "&apos;"), ("\"", "&quot;")]
+    
+    var key: String
+    var value: String?
+    var attributes: [String: String] = [:]
+    var children: [String: [_XMLElement]] = [:]
+    
+    internal init(key: String, value: String? = nil, attributes: [String: String] = [:], children: [String: [_XMLElement]] = [:]) {
```

Comment:
I think this could lose `internal`, which is the default anyway.

Reply:
Good point. I like to put `internal` to make the intention explicit (vs. "did I forget to make this `public/…`? 🤔"). But I forgot to make things consistent. I'm happy with removing it though.

Reply:
Fixed.